.github/workflows/publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish
+on:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 14.x
+          cache: "yarn"
+
+      - run: yarn
+      - run: yarn deps:build && yarn extension:compile
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        uses: changesets/action@v1
+        with:
+          publish: yarn release
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          VSCE_PAT: ${{ secrets.VSCE_PAT }}
```

Comment:
We probably also need the `GITHUB_TOKEN` here to create github releases etc

Reply:
Do I need to manually add the GITHUB_TOKEN to the secrets, or is it available?

Reply:
It's available by default, but you need to share it explicitly with the action by adding it to `env` here